Formula/lilypond.rb

```diff
@@ -0,0 +1,69 @@
+class Lilypond < Formula
+  desc "Music notation for everyone"
+  homepage "https://lilypond.org"
+  url "https://lilypond.org/download/sources/v2.22/lilypond-2.22.1.tar.gz"
+  sha256 "72ac2d54c310c3141c0b782d4e0bef9002d5519cf46632759b1f03ef6969cc30"
+  license "GPL-3.0-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "bison" => :build
+  depends_on "fontforge" => :build
+  depends_on "gettext" => :build
+  depends_on "glib" => :build
+  depends_on "imagemagick" => :build
+  depends_on "perl" => :build
+  depends_on "pkg-config" => :build
+  depends_on "t1utils" => :build
+  depends_on "texi2html" => :build
+  depends_on "texinfo" => :build
+  depends_on "texlive" => :build
+
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "ghostscript"
+  depends_on "guile@2"
+  depends_on "pango"
+  depends_on "python@3.9"
+
+  uses_from_macos "flex" => :build
+
+  resource "tex-gyre-fonts" do
+    url "https://mirrors.ctan.org/fonts/tex-gyre.zip"
+    sha256 "1773c470f9e388e087b68e3426e115af2cd236845a7e05ceb25b2a503409a7a3"
+  end
+
+  resource "urw-fonts" do
+    url "https://github.com/ArtifexSoftware/urw-base35-fonts/archive/20200910.tar.gz"
+    sha256 "e0d9b7f11885fdfdc4987f06b2aa0565ad2a4af52b22e5ebf79e1a98abd0ae2f"
+  end
+
+  def install
+    system "./autogen.sh", "--noconfigure"
```

Comment:
Are we actually supposed to run this? Usually it's only configure unless upstreams version is broken somehow.

Reply:
I removed this line, and consequently the autoconf dependency, and everything still working fine.